### PR TITLE
Update Airbyte docs with ClickHouse cloud integration example

### PR DESCRIPTION
**Pull request by guykoh**

### CODE REVIEW

**Comment by DanRoscigno** [comment]
marking as draft until the [dbt change is merged in Airbyte](https://github.com/airbytehq/airbyte/pull/14833).

File: docs/en/integrations/airbyte-and-clickhouse.md
Comment (DanRoscigno) [inline docs/en/integrations/airbyte-and-clickhouse.md:55]:
```suggestion
In order to use ClickHouse as a destination, the user you'll use need to have the permissions to create databases, tables and insert rows. We recommend creating a dedicated user for Airbyte (eg. `my_airbyte_user`) with the following permissions:

```sql
CREATE USER 'my_airbyte_user'@'%' IDENTIFIED BY 'your_password_here';
```

```

**Review by DanRoscigno — APPROVED** [review]
Made some suggestions.

File: docs/en/integrations/airbyte-and-clickhouse.md
Comment (DanRoscigno) [reply docs/en/integrations/airbyte-and-clickhouse.md:38]: sorry, missed that one.  I would rather not have the "start clickhouse" step up top since if they are using Cloud they do not have to start it, they just need the details.

**Comment by DanRoscigno** [comment]
@guykoh is this ready to be merged? I see that you closed your DBT PR in favor of another.

**Comment by guykoh** [comment]
I believe we can merge this pr, maybe let's ask Mark to review as well?

**Comment by mzitnik** [comment]
LGTM @DanRoscigno

**Comment by DanRoscigno** [comment]
@guykoh I don't have write access to your repo to resolve the conflicts, can you do it please?

**Pull request merged**
